pkg/volume/plugins.go

```diff
@@ -121,6 +122,9 @@ type VolumeHost interface {
 	// the provided spec.  See comments on NewWrapperBuilder for more
 	// context.
 	NewWrapperCleaner(spec *Spec, podUID types.UID, mounter mount.Interface) (Cleaner, error)
+
+	//Get cloud provider from kubelet
+	GetCloudProvider() cloudprovider.Interface
```

Comment:
If we change the interface, how many other plugins need to change to use the `GetCloudProvider` method instead of instantiating the cloud provider directly?